src/Components/Home.tsx

```diff
@@ -18,14 +30,123 @@ export default function Home({ user, setUser }: Props) {
     navigate("/login");
   };
 
+  // Mockup *Temporário*
+  const editais = {
+    em_andamento: [
+      {
+        id: 1,
+        nome: "Edital PGCOMP-09/2022",
+        etapa: { nome: "Inscrições Abertas", data_fim: "dd/mm/aaaa" },
+        inscrito: true,
+      },
+      {
+        id: 2,
+        nome: "Edital PGCOMP-07/2022",
+        etapa: {
+          nome: "Divulgação dos Resultados Parciais",
```

Comment:
```suggestion
          nome: "Divulgação dos Resultados Parciais",
```
Isso seria uma etapa?

Reply:
Sim, antes do recurso. Coloquei só pra dar exemplo de outra etapa. Tem um exemplo aqui https://pgcomp.ufba.br/edital-pgcomp-032022-processo-seletivo-para-concessao-de-bolsas-de-mestrado-e-doutorado-0